### PR TITLE
add function to check if domain metadata is locked

### PR DESCRIPTION
**Pull request by JamesEarle**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -104,6 +104,17 @@ export const createInstance = (config: Config): Instance => {
 
       return tx;
     },
+    isDomainMetadataLocked: async (
+      domainId: string,
+      signer: ethers.Signer
```
Comment (Remscar) [inline src/index.ts:109]: Why are you taking a signer? There is a web3 provider from the config

**Review by Remscar — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (JamesEarle) [reply src/index.ts:109]: I checked before adding this and there isn't?

image

Comment (Remscar) [reply src/index.ts]: I was confused with zauction sdk and it having a provider. Please add a provider to the config so we follow the same patterns.

**Review by Remscar — APPROVED** [review]

**Comment by Remscar** [comment]
Make sure to fix compiling errors

**Pull request merged**